sdk/core/azure-core/inc/azure/core/internal/http/pipeline.hpp

```diff
@@ -150,6 +150,20 @@ namespace Azure { namespace Core { namespace Http { namespace _internal {
               clientOptions.Transport));
     }
 
+    [[deprecated]]
```

Comment:
Let's put doxygen comment, I think using `@deprecated` command (https://www.doxygen.nl/manual/commands.html#cmddeprecated)

Reply:
I like the idea of using @deprecated. Fixing.